src/main/java/it/gov/pagopa/pu/organization/repository/OrganizationRepository.java

```diff
@@ -14,4 +19,17 @@ public interface OrganizationRepository extends JpaRepository<Organization, Long
   Optional<Organization> findByOrgFiscalCode(String orgFiscalCode);
   List<Organization> findByBrokerId(Long brokerId);
 
+  @Modifying
+  @Transactional
+  @RestResource(exported = false)
+  @Query("UPDATE Organization o SET o.ioApiKey = :apiKey WHERE o.id = :organizationId")
+  void updateIoApiKey(@Param("organizationId") Long organizationId, @Param("apiKey") byte[] apiKey);
+
+  @Modifying
+  @Transactional
+  @RestResource(exported = false)
+  @Query("UPDATE Organization o SET o.sendApiKey = :apiKey WHERE o.id = :organizationId")
+  void updateSendApiKey(@Param("organizationId") Long organizationId, @Param("apiKey") byte[] apiKey);
```

Comment:
Are these annotation really needed?
```suggestion
  void updateIoApiKey(Long organizationId, byte[] apiKey);

  @Modifying
  @Transactional
  @RestResource(exported = false)
  @Query("UPDATE Organization o SET o.sendApiKey = :apiKey WHERE o.id = :organizationId")
  void updateSendApiKey(Long organizationId, byte[] apiKey);
```